controllers/v1/programs.js

```diff
@@ -75,4 +75,186 @@ module.exports = class Programs extends Abstract {
     });
   }
 
+
+    /**
+  * @api {post} /kendra/api/v1/programs/create Create Program
+  * @apiVersion 1.0.0
+  * @apiName create
+  * @apiGroup programs
+  * @apiSampleRequest /kendra/api/v1/programs/create
+  * @apiHeader {String} internal-access-token internal access token  
+  * @apiHeader {String} X-authenticated-user-token Authenticity token 
+  * @apiParamExample {json} Request-Body:
+  * {
+      "externalId" : "PROGID01",
+      "name" : "DCPCR School Development Index 2018-19",
+      "description" : "DCPCR School Development Index 2018-19",
+      "isDeleted" : false,
+      "resourceType" : [ 
+          "program"
+      ],
+      "language" : [ 
+          "English"
+      ],
+      "keywords" : [],
+      "concepts" : [],
+      "createdFor" : [ 
+          "0126427034137395203", 
+          "0124487522476933120"
+      ],
+      "userId":"a082787f-8f8f-42f2-a706-35457ca6f1fd",
+      "imageCompression" : {
+          "quality" : 10
+      },
+      "components" : [ 
+          ObjectId("5b98fa069f664f7e1ae7498c")
+      ],
+      "scope" : {
+          "entityType" : "state",
+          "entities" : [ 
+              ObjectId("5d6609ef81a57a6173a79e78")
+          ],
+          "roles" : [
+              {
+                "_id" : "5d6e521066a9a45df3aa891e",
+                "code" : "HM"
+              }
+          ]
+      }
+    }
+  * @apiParamExample {json} Response:
+   {
+    "message": "Program created successfully",
+    "status": 200,
+    "result": {
+        "_id": "5ff09aa4a43c952a32279234"
+    }
+   } 
+  * @apiUse successBody
+  * @apiUse errorBody
+  */
+
+   /**
+   * Create program.
+   * @method
+   * @name create
+   * @param {Object} req - requested data.
+   * @returns {JSON} - created program document.
+   */
+
+  async create(req) {
```

Comment:
@aman-tunerlabs - Build a program list API that will have pagination and search by name, description, externalId